docs/modules/ROOT/pages/asciidoctor-api-options.adoc

```diff
@@ -115,6 +115,19 @@ OptionsBuilder optionsBuilder =
             Options.builder().templateDirs(new File("templates_path"));
 ----
 
+[[sourcemap]]
+== sourcemap
+
+Keeps track of the file and line number for each parsed block. (Useful for tooling applications where the association between the converted output and the source file is important).
```

Comment:
I think the parenthesis are a leftover?

Reply:
I copied this sentence as is from the Asciidoctor.js module to make sure that we don't provide deviating definitions. :D
But yeah, I'll remove the parens.